starky/src/config.rs

```diff
@@ -28,7 +37,7 @@ impl StarkConfig {
         }
     }
 
-    pub(crate) fn fri_params(&self, degree_bits: usize) -> FriParams {
+    pub fn fri_params(&self, degree_bits: usize) -> FriParams {
```

Comment:
Missing docs.

Reply:
Just a thought: The `fri_config.fri_params` is also `pub`. We want to minimize the public API we expose as much as possible. Could it be that we don't need both to be public?

Reply:
Given `hiding` (i.e. zero-knowledgeness) is hardcoded to `false` on the stark side, leaving users the possibility to wrongly call this method with `true` worries me a bit (as it could possibly happen), so I'd rather maintain the wrapper function at the stark level that obfuscates this parameter. Does that make sense?